### PR TITLE
Added Countries

### PR DESCRIPTION
**Pull request by klorpa**
* San Marino (Sammarinese)
* Marshall Islands (Marshallese)
* Syria (Syrian)
* Bermuda (Bermudian)
* Uruguay (Uruguayan)
* Monaco (Monégasque)
* Montenegro (Montenegrin)
* Cambodia (Cambodian)
* Cameroon (Cameroonian)
* Gabon (Gabonese)
* Djibouti (Djiboutian)
* Greenland (Greenlandic)
* Tuvalu (Tuvaluan)
* Zambia (Zambian)
* Albania (Albanian)
* Brunei (Bruneian)
* Singapore (Singaporean)

### CODE REVIEW

File: src/utility/summaryWidgets.tw [hunk 1]
```diff
@@ -894,6 +894,12 @@ will
 		<<set $args[0].summary += "Tuv">>
 	<<case "Zambian">>
 		<<set $args[0].summary += "Zam">>
+	<<case "Albanian">>
+		<<set $args[0].summary += "Bru">>
```
Comment (Frodfurgurn1991) [inline src/utility/summaryWidgets.tw]: How come they have the same abbreviation?

Comment (klorpa) [inline src/utility/summaryWidgets.tw]: Mistyped while copying / pasting. Thanks for finding this.

File: src/uncategorized/nationalityToAccent.tw [hunk 2]
```diff
@@ -392,6 +392,80 @@
...
+	<<if $language == "French">>
+	<<set $args[0].accent = 1>>
+	<<else>>
+	<<set $args[0].accent = _naturalAccent>>
+	<</if>>
+<<elseif ($activeSlave.nationality == "Montenegrin")>>
+	<<set $activeSlave.accent = $seed>>
+<<elseif ($activeSlave.nationality == "Cambodian")>>
+	<<set $activeSlave.accent = $seed>>
+<<elseif ($activeSlave.nationality == "Cameroonian">>
```
Comment (Folant) [inline src/uncategorized/nationalityToAccent.tw]: Missing closing bracket

Comment (Folant) [inline src/uncategorized/nationalityToAccent.tw]: ) missing

File: src/uncategorized/nationalityToAccent.tw [hunk 3]
```diff
@@ -392,6 +392,80 @@
...
+	<<else>>
+	<<set $activeSlave.accent = $seed>>
+	<</if>>
+<<elseif ($activeSlave.nationality == "Uruguayan")>>
+	<<if $language == "Spanish">>
+	<<set $activeSlave.accent = 1>>
+	<<else>>
+	<<set $activeSlave.accent = $seed>>
+	<</if>>
+<<elseif ($activeSlave.nationality == "Monégasque">>
```
Comment (Folant) [inline src/uncategorized/nationalityToAccent.tw]: ) missing

File: src/uncategorized/nationalityToAccent.tw [hunk 4]
```diff
@@ -392,6 +392,80 @@
...
+	<<set $activeSlave.accent = $seed>>
+<<elseif ($activeSlave.nationality == "Cambodian")>>
+	<<set $activeSlave.accent = $seed>>
+<<elseif ($activeSlave.nationality == "Cameroonian">>
+	<<if $language == "French">>
+	<<set $args[0].accent = 1>>
+	<<else>>
+	<<set $args[0].accent = _naturalAccent>>
+	<</if>>
+<<elseif ($activeSlave.nationality == "Gabonese">>
```
Comment (Folant) [inline src/uncategorized/nationalityToAccent.tw]: ) missing

File: src/uncategorized/nationalityToAccent.tw [hunk 5]
```diff
@@ -392,6 +392,80 @@
...
+	<<if $language == "French">>
+	<<set $args[0].accent = 1>>
+	<<else>>
+	<<set $args[0].accent = _naturalAccent>>
+	<</if>>
+<<elseif ($activeSlave.nationality == "Djiboutian">>
```
Comment (Folant) [inline src/uncategorized/nationalityToAccent.tw]: ) missing

File: src/uncategorized/nationalityToAccent.tw [hunk 6]
```diff
@@ -392,6 +392,80 @@
...
+	<<if $language == "French">>
+	<<set $args[0].accent = 1>>
+	<<else>>
+	<<set $args[0].accent = _naturalAccent>>
+	<</if>>
+<<elseif ($activeSlave.nationality == "Greenlandic")>>
+	<<set $activeSlave.accent = $seed>>
+<<elseif ($activeSlave.nationality == "Tuvaluan")>>
+	<<set $activeSlave.accent = $seed>>
+<<elseif ($activeSlave.nationality == "Zambian">>
```
Comment (Folant) [inline src/uncategorized/nationalityToAccent.tw]: ) missing

File: src/init/setupVars.tw [hunk 7]
```diff
@@ -353,6 +353,57 @@
...
+<<set setup.djiboutianSlaveSurnames = ["Abdallah", "Abdi", "Abdillahi", "Abdo", "Aden", "Bile", "Cabdi", "Cali", "Daher", "Dahir", "Daoud", "Deka", "Djama", "Elmi", "Farah", "Haji", "Hashi", "Hersi", "Houmed", "Houssein", "Hussein", "Ibrahim", "Idriss", "Ismael", "Ismail", "Issa", "Jama", "Mahamed", "Mahamoud", "Mohamed", "Mohamoud", "Moussa", "Nour", "Omar", "Osman", "Robleh", "Said", "Saleh", "Wais", "Youssouf", "Yusuf", "Yuusuf"]>>
+
+<<set setup.greenlandicSlaveNames = ["Aaja", "Aajunnguaq", "Agnethe", "Aka", "Ámak", "Amanda", "Augusta", "Dorte", "Dorthe", "Emma", "Erika", "Erna", "Gertrud", "Juliane", "Julie", "Kâlat", "Kâlîna", "Karla", "Kátalît", "Katrine", "Kîsta", "Kîstat", "Kîstîna", "Kítôra", "Kúngo", "Kûno", "Laakki", "Laannguaq", "Lãke", "Lãnguaĸ", "Lípíka, "Lippikka", "Maliina", "Mimi", "Naja", "Nikkuliinnguaq", "Nive", "Nivi", "Nukaaka", "Paarnannguaq", "Paarnaq", "Sara", "Sofie", "Tippu", "Tuula", "Uiloq", "Vivi"]>>
```
Comment (Folant) [inline src/init/setupVars.tw]: `"Lípíka,` missing "

File: src/uncategorized/nationalityToAccent.tw [hunk 8]
```diff
@@ -420,51 +420,51 @@
 	<</if>>
 <<elseif ($activeSlave.nationality == "Monégasque">>
 	<<if $language == "French">>
-	<<set $args[0].accent = 1>>
+	<<set $activeSlave.accent = = 1>>
```
Comment (Frodfurgurn1991) [inline src/uncategorized/nationalityToAccent.tw]: That's wrong.

**Pull request merged**